### PR TITLE
Add example aggressive recruiter response email

### PR DESCRIPTION
**Pull request by dougireton**
Add example aggressive recruiter response email. Identical to the I sent earlier.

### CODE REVIEW

File: recruiter-email.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Just wanted to reach out about your first CoffeeOps meet-up last Friday.
+
+Seattle CoffeeOps is a community-driven meet-up run by engineers. As such, we try to keep our meetings non-commercial and pitch-free. I'm sorry I didn't warn you of that when you and Ken(?) gave me your staffing pitch at the beginning of the meeting.
```
Comment (iennae) [inline recruiter-email.txt]: Is this supposed to be an example? maybe remove the name here? (I understand this is from a real email that was sent, just wanted to make sure of the intent for inclusion and whether it should have the name or not.)

**Comment by dougireton** [comment]
Thank you. I'll remove it.

File: recruiter-email.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<Name>,
+
+Just wanted to reach out about your first CoffeeOps meet-up last Friday.
+
+Seattle CoffeeOps is a community-driven meet-up run by engineers. As such, we try to keep our meetings non-commercial and pitch-free. I'm sorry I didn't warn you of that when you gave me your staffing pitch at the beginning of the meeting.
```
Comment (ctbarrett) [inline recruiter-email.txt]: This is perhaps a bit too specific to the last occurence. Perhaps re-wording to be a bit more flexible, something like this?

_**<As a reminder|As we discussed|As a head's up>,**_ Seattle CoffeeOps is a community-driven meet-up run by engineers. As such, we try to keep our meetings non-commercial and pitch-free.

File: recruiter-email.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<Name>,
+
+Just wanted to reach out about your first CoffeeOps meet-up last Friday.
```
Comment (ctbarrett) [inline recruiter-email.txt]: Suggestion:

Just wanted to reach out about _**<your first|the next>**_ CoffeeOps meet-up _**on \<date\>**_.

File: recruiter-email.txt [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+Seattle CoffeeOps is a community-driven meet-up run by engineers. As such, we try to keep our meetings non-commercial and pitch-free. I'm sorry I didn't warn you of that when you gave me your staffing pitch at the beginning of the meeting.
+
+We do welcome all folks (beginners and experienced folks) who want to actively contribute to building the community. We also have a couple of recruiters who attend regularly, listen, and participate in discussions.
```
Comment (ctbarrett) [inline recruiter-email.txt]: Suggestion:

We also have _**some recruiters and others in non-technical roles**_ who attend regularly, listen, and participate in discussions. _**There are many opportunities for networking, and forming meaningful relationships that may result in referrals/sales/follow-up outside of CoffeeOps, but we are a community first and foremost.**_

**Review by ctbarrett — APPROVED** [review]
Not sure how generic you want to make this, but here are a few ideas. Nothing necessarily blocking, though.

File: recruiter-email.txt (hunk 4 above)
Comment (dougireton) [reply recruiter-email.txt]: Excellent suggestions. I'll make these changes and resubmit.

**Review by gekitsuu — APPROVED** [review]
Agree with Craig's suggestions for the most part

**Comment by dougireton** [comment]
This is still on my radar, but I can't get to it before September 2017.

**Comment by prasket** [comment]
Are we waiting on anything to close this PR?

**Comment by dougireton** [comment]
Yes, we're waiting on me to incorporate Craig's changes. Thanks for checking in.

**Comment by dougireton** [comment]
I incorporated all suggestions above by @ctbarrett.

**Comment by johnharris85** [comment]
LGTM

**Pull request merged**